test/build/__snapshots__/build.test.js.snap

```diff
@@ -707,53 +761,102 @@ exports[`snowpack build legacy-mount-scripts: web_modules/import-map.json 1`] =
 }"
 `;
 
-exports[`snowpack build meta-dir: _dist_/index.js 1`] = `"console.log('I’m a console log! 🌲');"`;
```

Comment:
new: cleaned up existing meta-dir tests as well to trigger the same response for testing